Demo/cflink-simple.js

```diff
@@ -89,7 +89,7 @@ var CFL = {
 	_packetRegex: function(id,cmd,payload) {
 		id = (id == null || id === "") ? "[\\s\\S]" : "\\"+String.fromCharCode(parseInt(id,16));
 		payload = (payload == null || payload === "") ? "" : payload.replace("|","\\|");
-		return new RegExp("^\xF2" + id + "\xF3" + cmd + "\xF4" + payload);
+		return new RegExp("^\xF2" + '(' +id + '|\xFF)' + "\xF3" + cmd + "\xF4" + payload); // TODO: fix hack here around FF.
```

Comment:
This is ugly, but I'm not sure how else to handle the fact that the on-board serial port in a CFMini sends back FF instead of the CFLink ID. Maybe you can issue a firmware update?